apps/astarte_realm_management/lib/astarte_realm_management/engine.ex

```diff
@@ -937,6 +941,25 @@ defmodule Astarte.RealmManagement.Engine do
     end
   end
 
+  defp verify_mappings_max_storage_retention(realm_name, interface) do
+    with {:ok, max_retention} when is_integer(max_retention) <-
+           get_datastream_maximum_storage_retention(realm_name) do
+      if is_mappings_retention_valid?(interface.mappings, max_retention) do
```

Comment:
Functions with `?` should not start with `is`

Reply:
ops. I've found an "example" written by @bettio 
`Queries.is_interface_major_available?`
I'm removing it